packages/launchpad/cypress/e2e/project-setup.cy.ts

```diff
@@ -87,12 +87,10 @@ describe('Launchpad: Setup Project', () => {
     cy.contains('h1', 'Configuration files')
     cy.findByText('We added the following files to your project:')
 
-    cy.get('[data-cy=valid]').within(() => {
-      cy.contains('cypress.config.js')
-      cy.containsPath('cypress/support/e2e.js')
-      cy.containsPath('cypress/support/commands.js')
-      cy.containsPath('cypress/fixtures/example.json')
-    })
+    cy.get('[data-cy=valid]').as('valid').contains('cypress.config.js')
```

Comment:
Here's an example of where we were genuinely relying on multiple subjects from `.within()`, and it required something more than just `.first()` to fix the test.